_site/index.html

```diff
@@ -347,6 +349,45 @@ <h3>T-Z</h3>
           </div>
         </section>
 
+        <section>
+          <p class="word__title" id="bisexual">
+            Bisexual
+            <span class="word__speech">
+              adj
+            </span>
+          </p>
+          
+          <p class="word__definition">
+            someone who is sexually attracted to two or more genders.
```

Comment:
of, relating to, or characterized by being sexually attracted...

Reply:
What are your thoughts on using "attracted to more than one gender" and steering away from "two or more" since the biphobic rhetoric of "bi means two" is often used as a way to try to spread misinformation about bisexuality being inherently transphobic. I'd personally like to see bisexuality as far removed from the concept of "two" as much as possible.

Further, [Robin Ochs' widespread definition](https://biresource.org/bisexuality-101/), the [Bisexual Index](http://www.bisexualindex.org.uk/index.php/AmIBisexual), the [Human Rights Campaign](https://www.hrc.org/resources/glossary-of-terms), and even the [APA](https://www.apa.org/pi/lgbt/resources/bisexual) define bisexuality as "attraction to more than one gender" and not "two or more".

Reply:
I really love this point, Monika! I have long grappled with `bi` versus `pan` for myself for this reason (I disliked the bi means two rhetoric as a queer person, but as a grammarian understood it, even when I hated it).

Prposing we revise then, to:
`of, relating to, or characterized by being sexually attracted to more than one gender.`

Reply:
Agreed! Looking at those links, I think the one from the Bisexual Index is a good one to put in Further Reading. The one from biresource has some text on the interaction and divide between sex and gender which I know some trans people would take issue with.

Reply:
I think that's a great decision and I agree. My main goal for that resource was Ochs' quote but it looks like that's also outdated and she's updated the quote on [her personal site](https://robynochs.com/bisexual/) to focus multi-gender attraction.

Thanks for being open to this conversation with me, I appreciate it!